tests/tasks/cleanup.yml

```diff
@@ -0,0 +1,53 @@
+- hosts: all
+  tasks:
+    - name: Set platform/version specific variables
+      include_role:
+        name: linux-system-roles.vpn
+        tasks_from: set_vars
+        public: true
```

Comment:
nice - this seems to be an elegant solution to the problem of "I need to load the vars for the role outside of the role" - @pcahyna @nhosoi